app/controllers/audience_controller.rb

```diff
@@ -1,16 +1,31 @@
 # frozen_string_literal: true
 
 class AudienceController < Sellers::BaseController
-  before_action :set_body_id_as_app
-  before_action :set_time_range, only: %i[data_by_date]
+  before_action :set_time_range, only: :index
 
   after_action :set_dashboard_preference_to_audience, only: :index
   before_action :check_payment_details, only: :index
 
+  layout "inertia", only: :index
+
   def index
     authorize :audience
 
-    @total_follower_count = current_seller.audience_members.where(follower: true).count
+    total_follower_count = current_seller.audience_members.where(follower: true).count
+
+    render inertia: "Analytics/Audience/Index", props: {
+      total_follower_count:,
+      audience_data: InertiaRails.defer do
+        if total_follower_count.zero?
+          nil
+        else
+          CreatorAnalytics::Following.new(current_seller).by_date(
+            start_date: @start_date.to_date,
+            end_date: @end_date.to_date
+          )
+        end
+      end
```

Comment:
To ensure inertia doesn't load it on the first request. This helps in simplifying the frontend logic, allocating the first load concern to inertia and leaving subsequent to us